lib/bigshift/cli.rb

```diff
@@ -212,6 +215,8 @@ def rs_connection
         password: @config[:rs_credentials]['password'],
         sslmode: 'require'
       )
+      @rs_connection.exec("SET search_path = \"#{@config[:rs_schema_name]}\"")
```

Comment:
Is it possible to use `exec_params` and pass in the schema name as a variable?

I.e something like:

```ruby
@rs_connection.exec('SET search_path = $1', @config[:rs_schema_name])
```